### PR TITLE
Added bescort routine for ice road between Shard and Hib

### PR DESCRIPTION
**Pull request by adamgoerdt**

### CODE REVIEW

File: bescort.lic [hunk 1]
```diff
@@ -172,6 +178,7 @@ class Bescort
   def search
     fput('search')
     pause
```
Comment (rpherbig) [inline bescort.lic]: Instead of calling `pause` twice, you can pass it an arg (of seconds), e.g. `pause 1`.

File: bescort.lic [hunk 2]
```diff
@@ -359,6 +366,38 @@ class Bescort
...
     end
   end
+  
+  def iceroad(mode)
+    unless mode =~ /shard|hibarnhvidar/i
+      echo 'You must specify shard or hibarnhvidar for traversing the ice road'
+      exit
+    end
+    
+    hib_path = ['nw','ne','ne','e','ne','ne','ne','nw','sw','w','w','nw']
```
Comment (rpherbig) [inline bescort.lic]: I recommend declaring these two as constants (`$` at the top of the class).

File: bescort.lic (hunk 1 above)
Comment (adamgoerdt) [inline bescort.lic]: Not sure how that pause got in there.  I didn't intend to edit that part.  Must have been an errant keystroke which duplicated the line

File: bescort.lic (hunk 2 above)
Comment (adamgoerdt) [inline bescort.lic]: Updated

**Pull request merged**